scripts/README.md

```diff
@@ -0,0 +1,26 @@
+### Local environment tools
+Follow the steps below to work with the local Synthetic Monitoring cluster, and add a development version of the SM agent in there.
+
+First, setup a local cluster with [sm-dev](https://github.com/grafana/synthetic-monitoring-api/tree/main/scripts#sm-dev).
```

Comment:
This is fine, but since this repository is public, it probably makes sense to not assume that people have access to the sm-api repo.

Just leave it at something along the lines of "install the Synthetic Monitoring plugin in a Grafana instance and follow these instructions to add a new probe".

Note that the plugin reports the URL of the Synthetic Monitoring API that it's configured to use. That should be noted and used in the YAML below.

Reply:
Ohh you are right, didn't noticed that. Will do!

Reply:
Also inlined the `*.localhost` troubleshooting section.